### ISSUE TITLE
Invalid position given by 'W027'

### ISSUE THREAD

**Issue by spir1donov** [AtomLinter/linter-jshint#458]
JSHint returned a point that did not exist in the document being edited.
Rule: `W027`
Requested point: 219:16

Debug information:
```
Atom version: 1.18.0
linter-jshint version: v3.1.5
JSHint version: jshint v2.9.5
Hours since last Atom restart: 0.1
Platform: darwin
Current file's scopes: [
  "source.js",
  "constant.other.object.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

**Comment by Arcanemagus** [comment]
Is your code indented using tabs?

**Comment by spir1donov** [comment]
Yes it is. Is this the cause of the problem? 
Thanks!

**Comment by Arcanemagus** [comment]
It looks like this is another case of https://github.com/jshint/jshint/issues/3151 then. Please follow that issue for when this will be fixed in JSHint.

If anyone is directed here and _isn't_ using tabs to indent your file please file a new issue with a file to reproduce it on the JSHint repo, or file one here and I'll try to help you in getting one filed there where it can be fixed.

**Comment by spir1donov** [comment]
Yeah, that another case seems relevant.

Thank you anyway!

**Comment by Arcanemagus** [comment]
There are reports that if you set your `indent` property of your JSHint configuration to `1` this issue can be worked around. If you don't have a `.jshintrc` file the following should work:
```json
{
  "indent": 1
}
```

**Comment by spir1donov** [comment]
Hi! Thank you for your help, I really appreciate it

**Comment by Arcanemagus** [comment]
@spir1donov I'm actually leaving this issue open as a "catchall" for other users hitting this. If you don't want to receive notifications feel free to unsubscribe 😉.

**Comment by spir1donov** [comment]
Nice idea! I'll unsubscribe then
